contents/blog/what-motivates-me.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2022-10-07
+title: What motivates me
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+PostHog got pretty far (17k customers across all our products, went through Y Combinator, seed, series A and B raised, $MM revenue) before I really figured out what motivates me.
+
+## Before we started – building the _exact_ place I'd want to work
+
+I've worked at a bunch of startups before. They all had _many_ flaws.
+
+It is _a lot_ of fun getting things right, and this motivated me early in the company. These specifically stand out:
+
+* Writing stuff down in a [handbook](../handbook) rather than handling everything 1:1 in meetings. So much win.
+
+* [Sharing everything](how-to-run-a-transparent-company) internally and most of it externally too. And seeing how this helps people work autonomously.
+
+* Working on something for end users.
```

Comment:
This might be obvious to you, but what's the opposite to this? I think it's worth adding some clarity to what you mean / seek to avoid by focusing on end users.

Reply:
clarified this, thanks